azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,251 @@ def load_iothub_help():
+                az iot hub message-endpoint create cosmosdb-collection -n {iothub_name} --en {endpoint_name} -c {connection_string} --collection {collection} --db {database}
+          - name: Create a Cosmos DB Collection endpoint for an IoT Hub using the specified primary key and endpoint uri.
+            text: >
+                az iot hub message-endpoint create cosmosdb-collection -n {iothub_name} --en {endpoint_name} --pk {primary_key} --endpoint-uri {endpoint_uri} --collection {collection} --db {database}
+          - name: Create a Cosmos DB Collection endpoint for an IoT Hub using system assigned identity and a partition key name. The partition key template will be the default.
+            text: >
+                az iot hub message-endpoint create cosmosdb-collection -n {iothub_name} --en {endpoint_name} --endpoint-uri {endpoint_uri} --collection {collection} --db {database} --auth-type identityBased --pkn {partition_key_name}
+          - name: Create a Cosmos DB Collection endpoint for an IoT Hub using user assigned identity, partition key name, and partition key template.
+            text: >
+                az iot hub message-endpoint create cosmosdb-collection -n {iothub_name} --en {endpoint_name} --endpoint-uri {endpoint_uri} --collection {collection} --db {database} --auth-type identityBased --pkn {partition_key_name} --pkt {partition_key_template}
+    """
+
+    helps[
+        "iot hub message-endpoint create eventhub"
+    ] = """
+        type: command
+        short-summary: Add an Event Hub endpoint for an IoT Hub.
+        examples:
+          - name: Create an Event Hub endpoint for an IoT Hub using a connection string. The endpoint uri and entity path are omitted because they are in the connection string.
+            text: >
+                az iot hub message-endpoint create eventhub -n {iothub_name} --en {endpoint_name} -c {connection_string}
+          - name: Create an Event Hub endpoint for an IoT Hub using system assigned identity. The endpoint and entity path must be specified.
+            text: >
+                az iot hub message-endpoint create eventhub -n {iothub_name} --en {endpoint_name} --endpoint-uri {endpoint_uri} --auth-type identityBased --entity-path {entity_path}
+          - name: Create an Event Hub endpoint for an IoT Hub using user assigned identity. The endpoint and entity path must be specified.
+            text: >
+                az iot hub message-endpoint create eventhub -n {iothub_name} --en {endpoint_name} --endpoint-uri {endpoint_uri} --auth-type identityBased --entity-path {entity_path}
+    """
+
+    helps[
+        "iot hub message-endpoint create servicebus-queue"
+    ] = """
+        type: command
+        short-summary: Add a Service Bus Queue endpoint for an IoT Hub.
+        examples:
+          - name: Create a Service Bus Queue endpoint for an IoT Hub using a connection string. The endpoint uri and entity path are omitted because they are in the connection string.
```

Comment:
Is there not a variation of the endpoint commands where the CLI can fetch connection string or other details for the user ? As it is for DT endpoints.

Reply:
right now no but I can add that in

Reply:
Note that this will require a policy name that exists from users for eventhub/servicebus endpoints